public/index.html

```diff
@@ -3,12 +3,6 @@
   <head>
     <meta charset="utf-8" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Barlow:wght@400;500;700&family=Crimson+Text:wght@400&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
```

Comment:
tror man enten bruker MUI icons via fonter (som importert her), eller SVG fra `@material-ui/icons`, så tror ikke vi har hatt bruk for denne før heller. hvis det ikke stemmer er det uansett tilgjenglig fra npm: https://www.npmjs.com/package/@fontsource/material-icons

https://material-ui.com/components/icons/#font-vs-svg-which-approach-to-use